### ISSUE TITLE
ModMenu causes button to be offset incorrectly

### ISSUE THREAD

**Issue by Lncvrt** [The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher#157]
![image](https://github.com/The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher/assets/171465674/5e74f92b-c733-42c1-9d46-8d73eadbbfcf)

**Comment by VidTu** [comment]
that's definitely low-priority considering how ModMenu handles UI elements, especially in legacy versions

**Comment by Lncvrt** [comment]
This is 1.20.6. It's not legacy.

**Comment by VidTu** [comment]
i do understand this, however, this needs to be fixed in all versions, including 1.18.2, for example

**Comment by Lncvrt** [comment]
![image](https://github.com/The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher/assets/171465674/a57b1c56-0ed5-435e-a240-14b61a21489a)
This non default feature works with mod menu, but the default setting doesn't.
![image](https://github.com/The-Fireplace-Minecraft-Mods/In-Game-Account-Switcher/assets/171465674/48e09e39-fc72-4859-a8af-ddb847cce056)

So this shouldn't really be a low priority feature

**Comment by Lncvrt** [comment]
@VidTu any updates on when or if this has been fixed?

**Comment by VidTu** [comment]
No, this has NOT been fixed and for now, #185 will be main priority, because if I can't release (I won't be uploading 35 files manually) the mod, there will be no sense in developing this. I plan to fix this issue in the future, but you can help by making a PR.

@Lncvrt

**Comment by Lncvrt** [comment]
> No, this has NOT been fixed and for now, #185 will be main priority, because if I can't release (I won't be uploading 35 files manually) the mod, there will be no sense in developing this. I plan to fix this issue in the future, but you can help by making a PR.
> 
> @Lncvrt

Alright, I'll try and help if i do get time.
